tests/generic/check_seed.sql

```diff
@@ -0,0 +1,73 @@
+-- this tests checks a model for every row in a seed file.
+-- you need to specify the matching columns and the columns to check for equality.
+
+{% test check_seed(model, seed_file, match_columns=[], check_columns=[]) %}
+    {# --jinja comment
+--    I wish we could do something dynamic like this below, but it is prohibited by having a local database connection.. :(
```

Comment:
:(

Reply:
Looks like we only need 'read_metadata' permissions to do more fancy compile stuff. 😉

Reply:
I *think* that exposes s3 buckets so it's probably a no go. :(

Reply:
Just had a quick check, `DESCRIBE TABLE EXTENDED` does indeed expose data location. :(
https://docs.databricks.com/security/access-control/table-acls/object-privileges.html#operations-and-privileges
https://docs.databricks.com/spark/latest/spark-sql/language-manual/sql-ref-syntax-aux-describe-table.html#examples